### PR TITLE
Adds necessary files for hosting

### PR DESCRIPTION
**Pull request by loitijoseph**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -12,10 +16,25 @@ const notFoundMiddleware = require('./middleware/not-found');
...
+    windowMs: 15 * 60 * 1000,
+    max: 100,
+  })
+);
+app.use(express.json());
+app.use(helmet());
+app.use(cors());
+app.use(xss());
+
 app.use(express.static('./public'));
```
Comment (ChinyangaTL) [inline app.js:31]: Line 31 might need to change to this to get heroku to play nice

app.use(express.static(__dirname + './public'));

File: app.js [hunk 2]
```diff
@@ -12,10 +16,25 @@ const notFoundMiddleware = require('./middleware/not-found');
...
 app.use(express.json());
 
 // ROUTES
+app.get('/', (req, res) => {
+  res.send('Dear Diary API');
+});
```
Comment (ChinyangaTL) [inline app.js]: I want to deploy it with this entry point and without to see how the static files in the public will served so we'll test out both.

File: app.js (hunk 1 above)
Comment (ChinyangaTL) [reply app.js:31]: @loitijoseph forget about my previous comment. 
As we saw, it deployed nicely with the current format and the new one is breaking on my computer.
But as for lines 35 - 37

app.get('/', (req, res) => {
  res.send('Dear Diary API');
});

make sure you comment that out or delete it.

**Comment by optimusinhisprime** [comment]
Checked out the config and it looks proper. You'd already added the yml file so not much in terms of build server to be looked at.

File: app.js (hunk 1 above)
Comment (loitijoseph) [reply app.js:31]: Ok. Got it

File: app.js (hunk 2 above)
Comment (loitijoseph) [reply app.js]: No problem. Go ahead

**Comment by loitijoseph** [comment]
> Checked out the config and it looks proper. You'd already added the yml file so not much in terms of build server to be looked at.

Yep

**Pull request merged**